web/admin/application/controllers/KlearCustomTarificatorController.php

```diff
@@ -631,7 +631,31 @@ protected function retarificate($pks)
                 \Ivoz\Core\Infrastructure\Domain\Service\Cgrates\RerateCallService::class
             );
 
-            $rerateService->execute($pks);
+            try {
+                $rerateService->execute($pks);
+            } catch (\Exception $e) {
+
+                $close = $this->_helper->translate('Close');
+
+                $showErrors = ($e instanceof \DomainException) || (APPLICATION_ENV === 'development');
```

Comment:
Can this be extracted from the setting in application.ini ?

Reply:
Yes we can, but...

This is how it's done in klear:
 ```
        $phpSettings = $this->getInvokeArg('bootstrap')->getOption("phpSettings");
        $showErrors = false;
        if (array_key_exists("display_errors", $phpSettings) && $phpSettings["display_errors"]) {
            $showErrors = true;
        }
        if (APPLICATION_ENV == 'development' || $showErrors) {
            $showErrors = true;
        }
``` 

I mean, it's not been handled properly anywhere